projects/packages/jetpack-mu-wpcom/src/features/verbum-comments/class-verbum-comments.php

```diff
@@ -139,6 +151,8 @@ public function enqueue_assets() {
 			)
 		);
 
+		wp_enqueue_script( 'wp-i18n' );
```

Comment:
Is this necessary? I'm not sure why you've added it...

Reply:
I added a to-do to continue and investigate this. I think I'm close to figuring it out.

Reply:
Makes sense, thanks